json-guides/bulkhead.json

```diff
@@ -262,6 +262,18 @@
                                 "    Service chatService = new ChatSession(counterForVFA)",
                                 "    return chatService;",
                                 "  }",
+                                "",
+                                "  public Service requestForVFA() {",
```

Comment:
Move this above the serviceForVFA() method, so it matches the code in the previous steps